openshift/README.md

```diff
@@ -31,7 +31,9 @@ Starting with version 6.1, the Datadog Agent supports monitoring OpenShift Origi
 | Live Container monitoring      | ❌                                       | ❌                                           | ✅                                             |
 | Live Process monitoring        | ❌                                       | ❌                                           | ✅                                             |
 
-> :warning: **OpenShift 4.0+**: If you used the OpenShift installer on a supported cloud provider, you will need to deploy the Agent with `hostNetwork: true` to get host tags/aliases as access to metadata servers from PODs network is otherwise restricited.
+<div class="alert alert-warning">
+**OpenShift 4.0+**: If you used the OpenShift installer on a supported cloud provider, you will need to deploy the Agent with `hostNetwork: true` to get host tags/aliases as access to metadata servers from PODs network is otherwise restricited.
```

Comment:
```suggestion
**OpenShift 4.0+**: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with `hostNetwork: true` in the `datadog.yaml` configuration file to get host tags/aliases as access to metadata servers from PODs network, which otherwise is restricted.
```